### PR TITLE
docs(README): change and re-arrange README

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,34 +4,13 @@
...
+This library is being built from scratch in TypeScript using the Bootstrap 4 CSS framework. 
+This project is being created by the [ui-bootstrap team](https://github.com/angular-ui/bootstrap).
 
 As with Bootstrap 4 and Angular 2, this library is a work in progress. Please check out our list of
 [issues](https://github.com/ng-bootstrap/core/issues) to see all the things we are implementing.
 Feel free to make comments there.
 
-This project is being created by the [ui-bootstrap team](https://github.com/angular-ui/bootstrap).
-
-# Quick links
```
Comment (pkozlowski-opensource) [inline README.md]: We don't have that much content so I don't feel like we need those quick links for now.

File: README.md [hunk 2]
```diff
@@ -4,34 +4,13 @@
...
-
-- [Demo](#demo)
-- [Support](#support)
-    - [Code of Conduct](#code-of-conduct)
-    - [Supported browsers](#supported-browsers)
-- [Contributing to the project](#contributing-to-the-project)
-
-# Demo
```
Comment (pkozlowski-opensource) [inline README.md]: Demos are obsolete, no point in keeping them around.

File: README.md [hunk 3]
```diff
@@ -4,34 +4,13 @@
 [![devDependency Status](https://david-dm.org/ng-bootstrap/core/dev-status.svg?branch=master)](https://david-dm.org/ng-bootstrap/core#info=devDependencies)
 
 Welcome to the Angular 2 version of the [Angular UI Bootstrap](https://github.com/angular-ui/bootstrap) library.
-This library is being built from scratch in TypeScript using the Bootstrap 4 CSS framework.
+This library is being built from scratch in TypeScript using the Bootstrap 4 CSS framework. 
+This project is being created by the [ui-bootstrap team](https://github.com/angular-ui/bootstrap).
```
Comment (Foxandxss) [inline README.md]: With this we say twice the ui-bootstrap thing.

File: README.md [hunk 4]
```diff
@@ -45,7 +24,8 @@ See [this](https://github.com/angular/angular/blob/master/README.md) for up-to-d
 
 ## Contributing to the project
 
-**NOTE: We appreciate all the interest and enthusiasm, but this project is just in such a volatile state right now that we can't guarantee that your PR is going to be merged.
```
Comment (pkozlowski-opensource) [inline README.md]: I think that we no longer need to actively discourage contributions :-)

File: README.md (hunk 3 above)
Comment (icfantv) [inline README.md]: Yea, we can combine the two like:  "Welcome to the Angular 2 version of the ... library built by the ... team." or something similar.

**Comment by icfantv** [comment]
other than @Foxandxss' note, i'm good with this.

**Pull request closed without merging**